Gemfile.next.lock

```diff
@@ -149,7 +159,7 @@ GEM
     brakeman (5.4.1)
     buftok (0.2.0)
     builder (3.3.0)
-    bullet (7.0.1)
+    bullet (7.1.6)
```

Comment:
Upgrade `bullet` for Rails 7.1 support: https://github.com/flyerhzm/bullet/blob/main/CHANGELOG.md#710-10062023